### ISSUE TITLE
GFF-to-GBK removed

### ISSUE THREAD

**Issue by lecorguille** [vipints/GFFtools-GX#6]
Hi,

I'm wondering why you removed GFF-to-GBK from your tool list ?
https://github.com/vipints/GFFtools-GX/commit/c99274597ef7856e7b89d1afbbef33bd04295722

One of my users want to use something like that.

Thanks by advance

**Comment by vipints** [comment]
Sorry for the delay, I didn't see much need for a GBK file for handling the feature annotation back in that day and decided to retire the program from the tools list. are you still interested on this?

**Comment by yaanlpc** [comment]
of course, it's still very useful. like someone created a gene annotation in gff3, but the file can't be processed by the Geneious, so I need to convert it to gbk for a trial, in order to compare the gene annotation with known gbk downloaded from genBank (.gb).
